src/components/ProgressBar/index.tsx

```diff
@@ -124,22 +150,24 @@ function spinning(child: React.ReactChild) {
 export const ProgressBar: React.SFC<ProgressBarProps> = ({
   minimum = 0,
   maximum = 100,
-  value = 0,
+  value: propValue,
   title,
   description,
   animate,
   stops = [],
-  ...props
+  type = 'primary' as 'primary',
```

Comment:
Is this as really necessary? I think the original type of `type` already constraints the string to one of the two values, i.e., `primary` should be of the right type. Maybe I am wrong here.

Reply:
Without assigning default value `type`'s type is fine, but with the default value, it's changed to type string. Not sure how can I fix this.